composer.json

```diff
@@ -20,6 +20,7 @@
         "behat/transliterator": "^1.2",
         "symfony/console": "~2.5||~3.0||~4.0",
         "symfony/config": "~2.3||~3.0||~4.0",
+        "symfony/debug": "~2.1||~3.0||~4.0",
```

Comment:
... why ? as you're not using anything from this component. BTW, at least use supported versions such as `2.8`, `3.3`, ... and so on.

Reply:
Removed, not sure why I added that.